src/__test_resources__/aui5.0LegacyTestResources/widgetsFolderWithCalculatedMeasuresFrom2Cubes.ts

```diff
@@ -0,0 +1,117 @@
+export const widgetsWithCalculatedMeasuresFrom2Cubes = {
```

Comment:
It doesn't look like these two test files are being used yet.
If so, let's not add them in this PR.

Reply:
They were being used in `migratedCalculatedMeasuresInWidgets.ts`, I have now decided to combine both into one file instead.